<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchWidget</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#returnButton:!hover
{
border-radius:10px;
background-color:rgb(255,255,143);
}
#returnButton:hover
{
border-radius:10px;
background-color:rgb(255,100,143);
}</string>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>281</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="nameShowLabel">
      <property name="text">
       <string>NULL</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="salGradeLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>工薪等级：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="salGradeShowLabel">
      <property name="text">
       <string>NULL</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="jobLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>职位：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="jobShowLabel">
      <property name="text">
       <string>NULL</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="salaryLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>月薪：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="salaryShowLabel">
      <property name="text">
       <string>NULL</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#searchButton:!hover
{
border-radius:10px;
background-color:rgb(255,255,143);
}
#searchButton:hover
{
border-radius:10px;
background-color:rgb(255,100,143);
}</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="numberLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>编号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="numberrLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
